opendatahub-io/odh-dashboard
# Use username during nb spawning

Resolves: #564

## Description
This PR just adds the username to the enable notebook request to utilise features added in https://github.com/opendatahub-io/odh-dashboard/pull/645 and related PRs

## How Has This Been Tested?
1. Make sure your (admin) PV is present
2. Spawn notebook as impersonated user
3. Verify user's PV is claimed correctly
4. Notebook spawns successfully for targeted user

## Merge criteria:
